cmd/pipeline/deploy.go

```diff
@@ -116,15 +121,22 @@ func deploy(ctx context.Context) *cobra.Command {
 			if err := deployPipeline(ctx, name, namespace, repository, branch, filename, wait, timeout, variables); err != nil {
 				return err
 			}
+
+			pipelineURL, err := getPipelineURL(ctx, name, namespace)
+			if err != nil {
+				return err
+			}
 			if !wait {
 				log.Success("Pipeline '%s' scheduled for deployment", name)
+				log.Information("Follow your pipeline progress via the URL:\n    %s", pipelineURL)
 				return nil
 			}
 
 			if waitUntilRunning(ctx, name, namespace, timeout); err != nil {
 				return err
 			}
 			log.Success("Pipeline '%s' successfully deployed", name)
+			log.Information("Check your pipeline status via the URL:\n    %s", pipelineURL)
```

Comment:
@rberrelleza this is a deploy success with `--wait`, which sentence do you propose?